jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -193,6 +193,17 @@ def parameterized(harness_group: Iterable[Harness],
   ]]
 )
 
+lax_population_count = tuple(
+  Harness(f"{jtu.dtype_str(dtype)}",
+          lax.population_count,
+          [arg],
+          dtype=dtype)
+  for dtype in jtu.dtypes.all_integer + jtu.dtypes.all_unsigned
```

Comment:
It seems that the `population_count_p` in JAX works on booleans also (not sure what it does, perhaps returns 1 for True). We should test for boolean as well.

Reply:
This was actually [issue 3886](https://github.com/google/jax/issues/3886) which mattjj already fixed in [this commit](https://github.com/google/jax/commit/30980742c57f3b507d74b5ae79e9a75b5f57bc08), so JAX should not support booleans for `population_count` anymore.